pcsx2/Frontend/CommonHotkeys.cpp

```diff
@@ -43,7 +43,12 @@ void CommonHost::Internal::ResetVMHotkeyState()
 
 static void HotkeyAdjustTargetSpeed(double delta)
 {
-	const double min_speed = Achievements::ChallengeModeActive() ? 1.0 : 0.1;
+#ifdef ENABLE_ACHIEVEMENTS
```

Comment:
This bit shouldn't be needed - `ChallengeModeActive()` returns false when it's not compiled in: https://github.com/PCSX2/pcsx2/blob/280a41806f8a212597deac50a466fb052729c19e/pcsx2/Achievements.h#L60-L63